rpc_jobs/unit/re_unit_tests.yml

```diff
@@ -10,7 +10,7 @@
           trigger-phrase: '.*recheck_all.*|.*recheck_unit_?tests.*'
```

Comment:
Why is there a `?` in the trigger phrase?

Reply:
Fixed, this was my doing in 36f717f5 it seems. :(